### ISSUE TITLE
Simplify error type for syntax errors from "parser-16)

### ISSUE THREAD

**Issue by brettcannon** [microsoft/python-language-server#1350]
![image](https://user-images.githubusercontent.com/54418/61554719-71fc3c80-aa12-11e9-882a-4256c8770884.png)

Basically the "16" doesn't seem to make any specific sense and so I suspect it's an internal detail that's leaking out.

**Comment by MikhailArkhipov** [comment]
Basically same thing as in other languages: Language + error code. I.e. Python, parsing error, code 16. What would you want to see? Ex, C#

![image](https://user-images.githubusercontent.com/12820357/61555135-7ffe8d00-aa13-11e9-9889-a0ed01ca3772.png)

**Comment by brettcannon** [comment]
I guess for me personally I don't know as a user what to do with the "16" (initially I thought "what does the 16th parser do?" as if the parser was implemented in stages). Is there a table somewhere someone could reference to get more details on the error code(s)? Otherwise maybe "parser-error-code-16" or something along those lines?

**Comment by MikhailArkhipov** [comment]
Yeah, I don't particularly like it either. In fact, I am not sure user cares about error code either. Codes come handy if it is possible to suppress them with some pragmas, but we don't have that kind of functionality anyway. Perhaps do like TS:

![image](https://user-images.githubusercontent.com/12820357/61560130-46804e80-aa20-11e9-9343-e451bf785808.png)

`Unexpected token 'aaa' (Python parser, error code 16)
